life_game/game.rb

```diff
@@ -0,0 +1,151 @@
+class Game
+attr_accessor :board
+SIZE = 30
+
+  def initialize(board)
+    @board = board
```

Comment:
tutaj widzę dosyć spore ryzyko nazewnicze - bo to `@board` i `@board` którego używasz w metoda klasowych mają inne zasięgi.

Zmieniłbym nazwę - żeby nie mylić.